SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+We take security issues in our project seriously and appreciate the efforts of the community in improving the security of our project.
+
+## Supported versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x.x   | :white_check_mark: |
+| < 2.0   | :x:                |
+
+## Reporting a vulnerability
+
+To report a security issue, please email [allaboutolaf@frogpond.tech](mailto:allaboutolaf@frogpond.tech) with a description of the issue, the steps you took to create the issue, affected versions, and, if known, mitigations for the issue. Our team will respond within 3 working days of your email. If the issue is confirmed as a vulnerability, we will open a Security Advisory. This project follows a 90 day disclosure timeline.
```

Comment:
To make markdown copy more diffable, I like to ensure that each sentence is on a line. This should still render as one paragraph.

```suggestion
To report a security issue, please email [allaboutolaf@frogpond.tech](mailto:allaboutolaf@frogpond.tech) with a description of the issue, the steps you took to create the issue, affected versions, and, if known, mitigations for the issue.
Our team will respond within 3 working days of your email.
If the issue is confirmed as a vulnerability, we will open a Security Advisory.
This project follows a 90 day disclosure timeline.
```

Reply:
That's a great tip!